utils/__tests__/validate_install_plans.test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const { validateInstallPlanIds } = require('../validate-install-plan');
+
+const githubHelpers = require('../github-api-helpers');
+
+jest.mock('@actions/core');
+jest.spyOn(global.console, 'error').mockImplementation(() => {});
+
+jest.mock('../github-api-helpers', () => ({
+  ...jest.requireActual('../github-api-helpers'),
+  filterQuickstartConfigFiles: jest.fn(),
+}));
+
+const validQuickstartFilename = 'mock-quickstart-2/config.yml';
+const invalidQuickstartFilename1 = 'mock-quickstart-1/config.yml';
+const invalidQuickstartFilename2 = 'mock-quickstart-3/config.yml';
+
+const mockGithubAPIFiles = (filenames) =>
+  filenames.map((filename) => ({
+    sha: '',
+    filename: `utils/mock_quickstarts/${filename}`,
+    status: 'added',
+    additions: 0,
+    deletions: 0,
+    changes: 0,
+    blob_url: '',
+    raw_url: '',
+    contents_url: '',
+    patch: '',
+  }));
+
+describe('Action: validate install plan id', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  test('succeeds with valid install plan id', () => {
+    const files = mockGithubAPIFiles([validQuickstartFilename]);
+    githubHelpers.filterQuickstartConfigFiles.mockReturnValueOnce(files);
+
+    validateInstallPlanIds(files);
+    expect(global.console.error).not.toHaveBeenCalled();
+  });
+
+  test('fails with invalid install plan id', () => {
+    const files = mockGithubAPIFiles([invalidQuickstartFilename1]);
+    githubHelpers.filterQuickstartConfigFiles.mockReturnValueOnce(files);
+
+    validateInstallPlanIds(files);
+    expect(global.console.error).toHaveBeenCalledTimes(4);
```

Comment:
```suggestion
    expect(global.console.error).toHaveBeenCalled();
```

Is the number of calls important here, or could we just say `toHaveBeenCalled`?

The count feels a bit like a distraction. Think we're just saying, if there's any error the job will throw the error and fail the workflow check.

Reply:
I was following some other examples! and it should be called a specific number of times (for example, if there's two invalid ids that count would increase by 1) but i'm fine with removing it

Reply:
Ah good point on the other uses in the repo. Let's leave it as is then. 

>  it should be called a specific number of times

Especially considering there's some actual meaning to the number ^